sklearn/linear_model/least_angle.py

```diff
@@ -53,14 +53,14 @@ def lars_path(X, y, Xy=None, Gram=None, max_iter=500, alpha_min=0,
            ``None`` will be removed in v0.23. Use :func:`lars_path_gram`
            instead.
 
-    y : None or array-like shape=(n_samples,)
+    y : None or array-like of shape (n_samples,)
         Input targets.
 
-    Xy : array-like shape=(n_samples,) or (n_samples, n_targets), default=None
+    Xy : array-like of shape (n_samples,) or (n_samples, n_targets), default=None
```

Comment:
hm see my comment https://github.com/scikit-learn/scikit-learn/pull/12356/files#r311660392 I'm not sure I like this.

Reply:
ok seems like we're going with "of shape" now.